doc/fileformat/cable_cell.rst

```diff
@@ -1,7 +1,7 @@
 .. _formatcablecell:
 
-Arbor Cable Cell format (ACC)
-=============================
+Arbor Cable Cell
```

Comment:
We should at least mention ACC somewhere. It's a useful acronym.

Reply:
I removed it to have consistent-ish file format headings. Shall we mention the extensions in all format headings?

Reply:
Not necessarily. I would just like it mentioned somewhere, maybe in the body of the text?

Reply:
Sure! I can add it for all formats, maybe even have a matrix on the fileformat index.